src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Roboto;
+  src: url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+}
+
+.body {
+  margin: 20px;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  background: #fff;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
```

Comment:
move this rule to `html` tag selector

NOTE!
Move all common styles like rules you've applied to `html`, `body`, `*`, etc to `common.scss` file